pkg/cmd/roachtest/tests/awsdms.go

```diff
@@ -39,18 +41,24 @@ import (
 )
 
 const (
-	awsdmsWaitTimeLimit  = 1 * time.Hour
-	awsdmsUser           = "cockroachdbtest"
-	awsdmsDatabase       = "rdsdb"
-	awsdmsCRDBDatabase   = "defaultdb"
-	awsdmsCRDBUser       = "dms"
-	awsdmsNumInitialRows = 100000
+	awsdmsWaitTimeLimit       = 1 * time.Hour
+	awsdmsUser                = "cockroachdbtest"
+	awsdmsDatabase            = "rdsdb"
+	awsdmsCRDBDatabase        = "defaultdb"
+	awsdmsCRDBUser            = "dms"
+	awsdmsNumInitialRows      = 100000
+	awsRDSSecretName          = "rds!db-074de488-6274-4b3e-ad27-d008b8ffa750"
```

Comment:
how comfortable are we holding this secret information out on an open source repo?

Reply:
It should be fine given you need to have an IAM role that can access our AWS account. If someone from outside was able to do that we have problems.

Reply:
This password was going to be automatically rotated by secret manager until I learned this morning if you use postgres as a DMS endpoint, your password cant contain +,$ or ; and some of the generated passwords will contain those...

Reply:
can you a comment to that effect. can you also add a comment detailing that we keep this RDS instance up all the time?

also nit: can you put these `RDS` secrets in a separate const block and rename it to `awsrds...`